tests/test_script_output.py

```diff
@@ -0,0 +1,192 @@
+                                          infile])
+
+    hash = _calc_md5(open(outfile, 'rb'))
+    assert hash == 'e57be1402a9fd84d8481d8ef5a695596', hash
+
+
+def test_trim_low_abund_k21_M1e7_C2():
+    infile = utils.get_test_data('simple-genome-reads.fa')
+    outfile = utils.get_temp_filename('out')
+    utils.runscript('trim-low-abund.py', ['-C', '2', '-k', '21',
+                                          '-M', '1e7', '-o', outfile,
+                                          infile])
+
+    hash = _calc_md5(open(outfile, 'rb'))
+    assert hash == '21ad2f64ceeb1e153f1e513486df9434', hash
+
+
+def test_trim_low_abund_k21_M1e7_C3():
+    infile = utils.get_test_data('simple-genome-reads.fa')
+    outfile = utils.get_temp_filename('out')
+    utils.runscript('trim-low-abund.py', ['-C', '3', '-k', '21',
+                                          '-M', '1e7', '-o', outfile,
+                                          infile])
+
+    hash = _calc_md5(open(outfile, 'rb'))
+    assert hash == '3da84a42d8538e24240e78cd0eadf168', hash
+
+
+def test_trim_low_abund_k21_M1e7_C4():
+    infile = utils.get_test_data('simple-genome-reads.fa')
+    outfile = utils.get_temp_filename('out')
+    utils.runscript('trim-low-abund.py', ['-C', '4', '-k', '21',
+                                          '-M', '1e7', '-o', outfile,
+                                          infile])
+
+    hash = _calc_md5(open(outfile, 'rb'))
+    assert hash == '578f124937b5a639985cc0a808139653', hash
+
+
+def test_trim_low_abund_k21_M1e7_C4_variable():
+    infile = utils.get_test_data('simple-genome-reads.fa')
+    outfile = utils.get_temp_filename('out')
+    utils.runscript('trim-low-abund.py', ['-C', '4', '-k', '21',
+                                          '-V',
+                                          '-M', '1e7', '-o', outfile,
+                                          infile])
+
+    hash = _calc_md5(open(outfile, 'rb'))
+    assert hash == '154d2a25599b3dcd2cf9cd2375999e56', hash
+
+
+def test_trim_low_abund_k21_M1e7_C4_variable_Z25():
+    infile = utils.get_test_data('simple-genome-reads.fa')
+    outfile = utils.get_temp_filename('out')
+    utils.runscript('trim-low-abund.py', ['-C', '4', '-k', '21',
+                                          '-V', '-Z', '25',
+                                          '-M', '1e7', '-o', outfile,
+                                          infile])
+
+    hash = _calc_md5(open(outfile, 'rb'))
+    assert hash == 'eb8bc6ed6246fb819100b9e7dde6fad2', hash
+
+
+def test_trim_low_abund_k21_M1e7_C4_variable_Z15():
+    infile = utils.get_test_data('simple-genome-reads.fa')
+    outfile = utils.get_temp_filename('out')
+    utils.runscript('trim-low-abund.py', ['-C', '4', '-k', '21',
+                                          '-V', '-Z', '15',
+                                          '-M', '1e7', '-o', outfile,
+                                          infile])
+
+    hash = _calc_md5(open(outfile, 'rb'))
```

Comment:
This is nitpicking, but let's close this file after using it:

```python
with open(outfile, 'rb') as output:
    hash = _calc_md5(output)
```

(Bonus points if you avoid using `hash` as a variable name, since it is a built-in function)